packages/sharing-smart-contract/test/subscription.test.js

```diff
@@ -23,6 +25,103 @@ describe('Subscription.sol', () => {
     return [params.price, params.duration];
   }
 
+  describe('subscribeTo()', () => {
+    it('should allow a user to subscribe to a collection', async () => {
+      const { subscriptionContract, addr1 } = await loadFixture(deploySCFixture);
```

Comment:
Avoid duplicating code by extracting common setup steps into a beforeEach or before block.
```suggestion
      describe('subscribeTo()', () => {
  let subscriptionContract;
  let addr1;

  beforeEach(async () => {
    ({ subscriptionContract, addr1 } = await loadFixture(deploySCFixture));
    const tx = await subscriptionContract.connect(addr1).createCollection();
    await tx.wait();
  });
```

Reply:
I use loadFixture instead it's better because the same state for the  fork node will be the same at each beginning.